### DISCUSSION TITLE
How to make a request from server to a particular user?

### DISCUSSION THREAD

**Discussion by emanuelchristo** [feathersjs/feathers#2395] [Q&A]
I am using feathers with nuxt.js as a plugin and using the injected `client` in the vuex store as `this.$client`
```
import io from "socket.io-client";
import feathers from "@feathersjs/client";

const socket = io("http://192.168.1.35:3030");
const client = feathers();

client.configure(feathers.socketio(socket));
client.configure(feathers.authentication());

export default ({ app }, inject) => {
  inject("client", client);
};
```

I want to get the user state when an event happens on the server. How can I make a request from the server to user who is authenticated using `@feathersjs/authentication` local auth

**Comment by bwgjoseph** [comment]
Not quite sure what you are asking for exactly, but if you make a request to server from client like

```js
client.service('xyz').create({ ... });
```

In the server, you can grab the user info via [params.user](https://docs.feathersjs.com/api/services.html#params)

If you are connected via `socket`, you can make use of [channels](https://docs.feathersjs.com/api/channels.html) to send stuff to the client

If those are not what you are asking for, then do clarify what you actually need

**Reply by emanuelchristo** [reply]
Thank you for the response. I think channels is what I was looking for. I'll read up on it
